beetsplug/absubmit.py

```diff
@@ -164,8 +164,13 @@ def _get_analysis(self, item):
                     item=item, error=e
                 )
                 return None
+            decoded = bytearray()
```

Comment:
FWIW, this terminology tripped me up at first: in the Pythonverse, bytes are _encoded_ strings (i.e., they are abstract strings written down in bytes for a certain encoding). Unicode strings are _decoded_ strings. This naming is a bit backward.